test/client/fixtures/sandbox/node/attributes-test.js

```diff
@@ -529,6 +529,12 @@ test('anchor.toString()', function () {
     strictEqual(anchor.toString(), url);
 });
 
+if (window.Node.prototype.hasOwnProperty('attributes')) {
```

Comment:
The condition `window.Node.prototype.hasOwnProperty('attributes')` is not inditcator of `document.createTextNode('text').attributes === null`.

Let's use `browserUtils.isIE11` condition.